README.rst

```diff
@@ -14,18 +14,77 @@ For full details see http://figure.openmicroscopy.org
 
 
 Requirements
-------------
+============
 
-* OMERO 4.4.x or OMERO 5.x
+* OMERO 5.0.0 or later.
 
 Installation
-------------
+============
+
+Install OMERO.web.
+
+This app installs into the OMERO.web framework.
+
+::
 
-Please see instructions at http://figure.openmicroscopy.org
+    $ pip install omero-figure
 
+Add figure custom app to your installed web apps:
+
+::
+
+    $ bin/omero config append omero.web.apps '"omero_figure"'
+
+Display a link to 'Figure' at the top of the webclient:
+
+::
+
+    $ bin/omero config append omero.web.ui.top_links '["Figure", "figure_index", {"title": "Open Figure in new tab", "target": "figure"}]' 
+
+Now restart OMERO.web as normal.
+
+**Warning**:
+
+if OMERO.figure is installed with OMERO version prior to **5.2.6**,
+the url will be https://your-web-server/omero_figure instead of https://your-web-server/figure as previously. This is due to a package re-organization required to distribute the application using a package manager.
+If installed with OMERO **5.2.6 and older**, the url will be back to https://your-web-server/figure.
+
+
+Enabling PDF generation
+-----------------------
+
+This following steps need to perform where the OMERO.server is running.
+
+* Install `reportlab <https://bitbucket.org/rptlab/reportlab>`_ PDF python package:
+
+::
+
+    $ pip install reportlab
+
+**Note**: For python 2.6, you will need Reportlab 2.7
+
+
+* Install script to generate figure. To perform this task, you must be an admin.
+
+The following steps assume that the OMERO.server is already running.
+
+- Go to the directory where ``omero_figure`` has been installed, this could be on a different machine.
```

Comment:
This is kinda confusing. I think it needs more explanation since the concept of "installing" on one machine while "on a different machine" is tricky, especially as you've just said "The following steps need to performed where the OMERO.server is installed".
If users have already set up OMERO.web on a different machine from OMERO.server, then I don't think we need to emphasise that "this could be on a different machine".
I also think it might be clearer to describe the script install first, then deal with dependencies. Here's a suggestion for the whole section:


Enabling figure export
----------------------

Figures are exported as PDF or TIFF files using a script that runs on the OMERO.server. This script needs to be uploaded to the OMERO.server and its dependencies installed on the OMERO.server machine.

The script can be uploaded using two alternative workflows, both of which require you to be an admin.

1) Connect to the OMERO server and upload script via the CLI. It is important to be in the correct directory when uploading so that the script is uploaded with the full path: ``omero/figure_scripts/Figure_To_Pdf.py``

    - $ cd omero_figure/scripts
    - $ path/to/OMERO.server/bin/omero script upload omero/figure_scripts/Figure_To_Pdf.py --official

 2) Alternatively, before starting the OMERO.server, copy the script from the figure install
``/omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py`` to the OMERO.server ``path/to/OMERO.server/lib/scripts/omero/figure_scripts``. Then restart the server.


Install reportlab....

Installl markdown...